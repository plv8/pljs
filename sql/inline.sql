DO $$ pljs.elog(NOTICE, 'this', 'is', 'inline', 'code') $$ LANGUAGE pljs;
DO $$ pljs.return_next(new Object());$$ LANGUAGE pljs;
