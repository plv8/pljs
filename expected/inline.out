DO $$ pljs.elog(NOTICE, 'this', 'is', 'inline', 'code') $$ LANGUAGE pljs;
NOTICE:  this is inline code
DO $$ pljs.return_next(new Object());$$ LANGUAGE pljs;
